### ISSUE TITLE
[Bug]: Accordion Keyboard Navigation

### ISSUE THREAD

**Issue by severinlandolt** [tremorlabs/tremor-npm#822]
### Tremor Version

3.11.0+

### Link to minimal reproduction

-

### Steps to reproduce

The accordion is not triggerable using the keyboard, this should be improved.

### What is expected?

Accordion should be tabable and open/close using the keyboard

### What is actually happening?

No feedback

**Comment by shreyas0924** [comment]
Hey.. After fixing the bug and writing test, how do I check whether the feat is working or not. Please help me out

**Comment by severinlandolt** [comment]
Hey @shreyas0924! You can check it out using storybook →  `npm run storybook`

**Comment by shreyas0924** [comment]
Thanks @severinlandolt

**Comment by shreyas0924** [comment]
@severinlandolt I did not find any issue with the Accordian. Its just that the outline is not visible. Otherwise the component is tabbable.
